packages/subgraph/tests/dao/dao.test.ts

```diff
@@ -60,251 +88,245 @@ test('Run dao (handleNativeTokenDeposited) for native token mappings with mock e
+  assert.fieldEquals('NativeTransfer', transferId, 'type', 'Deposit');
 
   clearStore();
 });
 
-test('Run dao (handleExecuted) for Token mappings with mock event', () => {
-  // create state
-  let daoEntity = Address.fromString(DAO_ADDRESS).toHexString();
-  createDaoEntityState(daoEntity, ADDRESS_ONE, DAO_TOKEN_ADDRESS);
-
-  let proposalId =
-    Address.fromHexString(CONTRACT_ADDRESS).toHexString() + '_' + ZERO_BYTES32;
-
-  createTokenVotingProposalEntityState();
-
-  // create token calls
-  createTokenCalls(DAO_TOKEN_ADDRESS, 'DAO Token', 'DAOT', '6');
-  getBalanceOf(DAO_TOKEN_ADDRESS, DAO_ADDRESS, HALF_ETH);
-
-  // create event
-  let callData =
-    '0x4f0656320000000000000000000000006b175474e89094c44da98b954eedeac495271d0f0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000014536f6d6520537472696e672044617461202e2e2e000000000000000000000000';
-  let actions = createDummyActions(DAO_ADDRESS, '0', callData);
-  let event = createNewExecutedEvent(
-    Address.fromHexString(CONTRACT_ADDRESS).toHexString(),
-    ZERO_BYTES32,
-    actions,
-    [Bytes.fromUTF8('')],
-    Address.fromHexString(DAO_ADDRESS).toHexString()
-  );
-
-  // handle event
-  handleExecuted(event);
-
-  let withDrawParams = decodeWithdrawParams(
-    Bytes.fromHexString('0x' + callData.slice(10))
-  );
-  // checks
-  let entityID =
-    Address.fromHexString(DAO_ADDRESS).toHexString() +
-    '_' +
-    event.transaction.hash.toHexString() +
-    '_' +
-    event.transactionLogIndex.toHexString() +
-    '_' +
-    withDrawParams.to.toHexString() +
-    '_' +
-    withDrawParams.amount.toString() +
-    '_' +
-    withDrawParams.token.toHexString() +
-    '_' +
-    withDrawParams.reference;
-
-  assert.fieldEquals('VaultTransfer', entityID, 'id', entityID);
-  assert.fieldEquals(
-    'VaultTransfer',
-    entityID,
-    'dao',
-    Address.fromHexString(DAO_ADDRESS).toHexString()
-  );
-  assert.fieldEquals(
-    'VaultTransfer',
-    entityID,
-    'token',
-    Address.fromHexString(DAO_TOKEN_ADDRESS).toHexString()
-  );
-  assert.fieldEquals(
-    'VaultTransfer',
-    entityID,
-    'to',
-    Address.fromHexString(ADDRESS_ONE).toHexString()
-  );
-  assert.fieldEquals('VaultTransfer', entityID, 'amount', '1');
-  assert.fieldEquals(
-    'VaultTransfer',
-    entityID,
-    'reference',
-    Bytes.fromUTF8(STRING_DATA).toString()
-  );
-  assert.fieldEquals('VaultTransfer', entityID, 'proposal', proposalId);
-  assert.fieldEquals(
-    'VaultTransfer',
-    entityID,
-    'transaction',
-    event.transaction.hash.toHexString()
-  );
-  assert.fieldEquals(
-    'VaultTransfer',
-    entityID,
-    'createdAt',
-    event.block.timestamp.toString()
-  );
-
-  clearStore();
-});
+// test('Run dao (handleDeposited) for Token mappings with mock event', () => {
```

Comment:
Do we have a follow-up task to add these tests?

Reply:
yep. working on it.